src/Bash/Base.hs

```diff
@@ -74,15 +77,24 @@ toNamespace (_:fs) = toNamespace fs
 -- | Never call this directly. Use `value` in `Aura/Bash`.
 getVar :: Namespace -> String -> Maybe [String]
 getVar ns s = case M.lookup s ns of
-                Nothing -> Nothing
-                Just bs -> Just $ map fromBashString bs
-
-fromBashString :: BashString -> String
-fromBashString (SingleQ s) = s
-fromBashString (DoubleQ s) = s
-fromBashString (NoQuote s) = s
-fromBashString (Backtic c) = '`' : fromCommand c ++ "`"
-
-fromCommand :: Field -> String
-fromCommand (Command c as) = unwords $ c : map fromBashString as
+  Nothing -> Nothing
+  Just bs -> Just $ foldr (++) [] $ map fromBashString bs
+
+fromBashString :: BashString -> [String]
+fromBashString (SingleQ s) = [s]
+fromBashString (DoubleQ l) = fromBashStringList l
+fromBashString (NoQuote l) = fromBashStringList l
+fromBashString (Backtic c) = ['`' : unwords (fromCommand c) ++ "`"]
+
+fromBashStringList :: [Either BashExpansion String] -> [String]
```

Comment:
You can probably just get rid of that function altogether, and just call `rights` directly.